### PR TITLE
Publish staged changes

### PR DESCRIPTION
**Pull request by armstrongl**
**Select the type of change:**

- [x] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**

**Issue Number:**

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=951) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=951) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=951) it.</sub>

File: docs/advanced-topics/message-system/clientrpc.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ darkImageSrc="/img/sequence_diagrams/RPCs/ClientRPCs_Dark.png?text=DarkMode"/>
 
 ## Declaring a ClientRpc
 
-You can declare a `ClientRpc` by marking a method with `[ClientRpc]` attribute and including the `ClientRpc` suffix in the method name.
+You can declare a ClientRpc by marking a method with `[ClientRpc]` attribute and including the `ClientRpc` suffix in the method name.
```
Comment (NoelStephensUnity) [inline docs/advanced-topics/message-system/clientrpc.md:20]: > method with **the** `[ClientRpc]`
?

File: docs/advanced-topics/message-system/clientrpc.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ void PongClientRpc(int somenumber, string sometext) { /* ... */ }
 
 ## Invoking a ClientRpc
 
-You can invoke a `ClientRpc` by making a direct function call with parameters:
+You can invoke a ClientRpc by making a direct function call with parameters:
```
Comment (NoelStephensUnity) [inline docs/advanced-topics/message-system/clientrpc.md:29]: > You can invoke a ClientRpc by invoking the function directly with parameters:
?

File: docs/advanced-topics/message-system/clientrpc.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ void Pong(int somenumber, string sometext) { /* ... */ }
 void PongClientRpc(int somenumber, string sometext) { /* ... */ }
 ```
 
-The `[ClientRpc]` attribute and matching `...ClientRpc` suffix in the method name are there to ensure RPC call sites know when they're executing an RPC. The client RPC replicates and executes client-side, without jumping into the original RPC method declaration to find out if it's an RPC.
+The `[ClientRpc]` attribute and matching `...ClientRpc` suffix in the method name ensure RPC call sites know when they're executing an RPC. The client RPC replicates and executes client-side, without jumping into the original RPC method declaration to find out if it's an RPC.
```
Comment (NoelStephensUnity) [inline docs/advanced-topics/message-system/clientrpc.md:52]: > The `[ClientRpc]` attribute and matching `...ClientRpc` suffix in the method name help to assure both context clarity in scripts that invoke them and are used by Netcode during the ILPostProcessor pass.  The ILPostProcessor pass replaces all call-site locations, where the RPC method is invoked, with additional code generated specific to the RPC to assure that the RPC message is generated and sent to the appropriate destinations/targets.

Something like the above I think might work?

File: docs/advanced-topics/message-system/custom-messages.md [hunk 4]
```diff
@@ -5,14 +5,14 @@ description: A brief explanation of Custom Messages use in Netcode for GameObjec
...
+If you don't want to use the Netcode for GameObjects (Netcode) messaging system, you don't have to. You can use a thin layer called "Custom Messages" to implement your own messaging behavior or add custom targeting. They're unbound to any GameObject. You can use Custom messages with [RPC messages](../messaging-system.md). 
 
 There are two types of custom messages:
 - Unnamed
 - Named
 
 ## Unnamed Messages
-Unnamed messages can be thought of as a single sending channel. A message sent has one receive handler, this is useful for building your own custom messaging system. Netcode for GameObjects handles delivering and receiving custom unnamed messages, you can provide your own "custom message headers" to determine the type of unnamed message being sent or received.
+You can think about named messages as a single sending channel. A message sent has one receive handler, which is useful for building your own custom messaging system. Netcode for GameObjects handles delivering and receiving custom unnamed messages. You can create your own custom message headers to find the unnamed message types.
```
Comment (NoelStephensUnity) [inline docs/advanced-topics/message-system/custom-messages.md:15]: > You can think about unnamed messages as if you are sending information over a single unique channel.  There is only one receiver handler per unnamed message, which can useful for building your own custom messaging system where you can define your own message headers. Netcode for GameObjects handles delivering and receiving custom unnamed messages, you determine what kind of information you want to transmit over the channel.

Maybe something like the above?

File: docs/advanced-topics/message-system/clientrpc.md (hunk 1 above)
Comment (armstrongl) [reply docs/advanced-topics/message-system/clientrpc.md:20]: This wasn't new content. Does this not make sense? Maybe we should remove it?

File: docs/advanced-topics/physics.md [hunk 5]
```diff
@@ -8,26 +8,26 @@ There are many different ways to do physics in multiplayer games. Netcode for Ga
 
 ## NetworkRigidbody
 
-`NetworkRigidbody` is a component which sets the `Rigidbody` of the `GameObject` into kinematic mode on every peer except the server. That way a realistic physics simulation can be run on the server and the resulting positions can be synchronized to the clients without any interference.
+NetworkRigidbody is a component that sets the Rigidbody of the GameObject into kinematic mode on every peer (except the server). That way, a realistic physics simulation runs on the server and the resulting positions synchronize to the clients without any interference.
```
Comment (NoelStephensUnity) [inline docs/advanced-topics/physics.md:11]: > NetworkRigidbody is a component that sets the Rigidbody of the GameObject into kinematic mode on all non-authoritative instances (except the instance that has authority). Authority is determined by the NetworkTransform component (required) that is attached to the same GameObject as the NetworkRigidbody. Whether the NetworkTransform is server authoritative (default) or owner authoritative, the NetworkRigidBody authority model will mirror it.  That way, the physics simulation runs on the authoritative instance and the resulting positions synchronize on the non-authoritative instances, each with their RigidBody being Kinematic, without any interference.

How does the above sound?

File: docs/advanced-topics/serialization/inetworkserializable.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ struct MyComplexStruct : INetworkSerializable
 }
 ```
 
-Types implementing `INetworkSerializable` are supported by `NetworkSerializer` , `RPC` s and `NetworkVariable` s.
+Types implementing `INetworkSerializable` are supported by `NetworkSerializer`, `RPC` s and `NetworkVariable` s.
```
Comment (NoelStephensUnity) [inline docs/advanced-topics/serialization/inetworkserializable.md:27]: > Types implementing `INetworkSerializable` are supported by `NetworkSerializer`, `RPC`s and `NetworkVariable`s

Just removing some spaces.

File: docs/basics/connection-approval.md [hunk 7]
```diff
@@ -1,37 +1,37 @@
...
 
-Connection approval allows you to decide, on a per connection basis, if the connection should be allowed. Connection approval also enables you to specify the player Prefab to be created, allowing you to override the default `NetworkManager` defined player Prefab on a per player basis. By setting `ConnectionApproval` property of the `NetworkManager` to `true` Netcode will then check to make sure the `NetworkManager.ConnectionApprovalCallback` has been assigned. If it has been assigned, then Netcode will use the connection approval process for connecting clients. 
+Connection approval allows you to decide, on a per connection basis, whether to allow a connection. Connection approval also enables you to specify the player Prefab to create, allowing you to override the default NetworkManager defined player Prefab on a per player basis. By setting ConnectionApproval property of the NetworkManager to true, Netcode checks to make sure the `NetworkManager.ConnectionApprovalCallback` has been assigned. If assigned, Netcode uses the connection approval process for connecting clients.
```
Comment (NoelStephensUnity) [inline docs/basics/connection-approval.md:8]: > ...whether to allow a connection or deny it.

? Possibly ?

**Review by NoelStephensUnity — APPROVED** [review]
Just a few more minor possible suggestion...
Really curious about how we got the "isn'tified" in so many places...
But everything else looks good to me!

File: docs/advanced-topics/message-system/clientrpc.md (hunk 3 above)
Comment (armstrongl) [reply docs/advanced-topics/message-system/clientrpc.md:52]: Love it!

File: docs/advanced-topics/message-system/custom-messages.md (hunk 4 above)
Comment (armstrongl) [reply docs/advanced-topics/message-system/custom-messages.md:15]: Much better. I'll just make a few tweaks  🙂. 

> You can think about unnamed messages as if you are sending information over a single unique channel. There's only one receiver handler per unnamed message, which can help when building a custom messaging system where you can define your own message headers. Netcode for GameObjects handles delivering and receiving custom unnamed messages; you determine what kind of information you want to transmit over the channel.

File: docs/basics/connection-approval.md (hunk 7 above)
Comment (armstrongl) [reply docs/basics/connection-approval.md:8]: I think it helps 🙂

File: docs/basics/connection-approval.md [hunk 8]
```diff
@@ -120,9 +122,9 @@ public class ConnectionApprovalHandler : MonoBehaviour
 ```
 
 
-## Connection data  (_`NetworkManager.ConnectionApprovalRequest.Payload`_)
+## Connection data (`NetworkManager.ConnectionApprovalRequest.Payload`)
 
-The `ConnectionApprovalRequest.Payload` parameter takes any custom data of your choice that the client should send to the server. Usually, this data should be some sort of ticket, room password, or similar that will decide if a connection should be approved or not. The `connectionData` is specified on the Client-side in the `NetworkingConfig` supplied when connecting.
+The `ConnectionApprovalRequest.Payload` parameter takes any custom data the client should send to the server. Usually, this data should be some sort of ticket, room password, or similar that helps the server decide whether to approve a connection. The `connectionData` is specified on the client-side in the `NetworkingConfig` (supplied when connecting).
```
[the remark this answers is not in the document]
Comment (armstrongl) [reply docs/basics/connection-approval.md:127]: I think this helps a lot! 😀

**Pull request merged**